core/iwasm/interpreter/wasm_loader.c

```diff
@@ -8116,6 +8134,8 @@ wasm_loader_ctx_destroy(WASMLoaderContext *ctx)
         if (ctx->frame_csp_bottom) {
 #if WASM_ENABLE_FAST_INTERP != 0
             free_all_label_patch_lists(ctx->frame_csp_bottom, ctx->csp_num);
+            free_all_label_param_frame_offsets(ctx->frame_csp_bottom,
```

Comment:
wasm_loader_pop_frame_csp will also free the param_frame_offsets, should we set it to NULL in wasm_loader_pop_frame_csp?

Reply:
The wasm_loader_push_frame_csp will memset the frame csp with 0, seems no need to set param_frame_offsets to NULL again:
https://github.com/bytecodealliance/wasm-micro-runtime/blob/67638e24f40a537b194a233a2d388f06db6d9546/core/iwasm/interpreter/wasm_loader.c#L8672-L8678

Reply:
Got it, thanks!